oga/frontend/src/store/actions/answerActions.js

```diff
@@ -0,0 +1,27 @@
+import * as actionTypes from './actionTypes.js';
+import axios from 'axios';
+
+import { push } from 'connected-react-router';
+
+axios.defaults.xsrfCookieName="csrftoken";
+axios.defaults.xsrfHeaderName="X-CSRFTOKEN";
+
+export const createAnswer_ = (answer) => {
+  return {
+    type: actionTypes.CREATE_ANSWER,
+    question_id: answer.question_id,
+    author_id: answer.author_id,
+    question_type: answer.question_type,
+    answer_content: answer.answer_content,
```

Comment:
can't we just use content, instead of answer_content. I think it's just redundant.